ping_app/notifier_service.py

```diff
@@ -20,9 +20,12 @@ def get_changed_state(self, instance: Zone | Host) -> str:
 
     @staticmethod
     def _calculate_hours_and_minutes(instance: Zone | Host) -> str:
-        seconds_since_last_update = (datetime.now() - instance.updated_at).seconds
+        time_since_last_update = datetime.now() - instance.updated_at
+        seconds_since_last_update = time_since_last_update.seconds
+        days: int = time_since_last_update.days
+        days_str = str(days) + ' днів' if days else ''
```

Comment:
```suggestion
        days_str = str(days) + ' днів ' if days else ''
```

Reply:
Intentionally I suppose yo use some sort of ruby-like
```ruby
{ days:, hours:, seconds: }.reject { <exclude zero values> }.map { <transform to strings> }.join(' ')
```
but since I have lack of python experience just stick to style it was here